constraint/pkg/client/drivers/local/local.go

```diff
@@ -78,6 +79,7 @@ type Driver struct {
 	printHook     print.Hook
 	providerCache *externaldata.ProviderCache
 	externs       []string
+	handlers      map[string]client.TargetHandler
```

Comment:
Add a comment that this is a map from handler name to the respective handler.

Reply:
done